telegram/_bot.py

```diff
@@ -654,14 +639,20 @@ async def _do_post(
 
         request = self._request[0] if endpoint == "getUpdates" else self._request[1]
 
-        return await request.post(
+        self._LOGGER.debug("Calling Bot API endpoint `%s` with parameters `%s`", endpoint, data)
```

Comment:
That's right. Thought it might be useful in terms of validating the result against parameters that are set at runtime. But it's not a strict must-have for me.

Reply:
I'm leaning toward removing the logging of parameters, but keeping the end result logging.

Reply:
I'll calculate this +0 and -0 as 0 here and keep the changes as is. If we get a notable number of complaints about too verbose logging, we can still remove it …